tests/integration/components/trigger-test.js

```diff
@@ -57,17 +55,17 @@ module('Integration | Component | basic-dropdown-trigger', function (hooks) {
       .hasAttribute('tabindex', '0', 'Has a tabindex of 0');
   });
 
-  test('If it receives a tabindex={{false}}, it removes the tabindex', async function (assert) {
-    assert.expect(1);
-    this.dropdown = { uniqueId: 123 };
-    await render(hbs`
-      <BasicDropdownTrigger tabindex={{false}} @dropdown={{this.dropdown}}>Click me</BasicDropdownTrigger>
-    `);
+  // test('If it receives a tabindex={{false}}, it removes the tabindex', async function (assert) {
+  //   assert.expect(1);
+  //   this.dropdown = { uniqueId: 123 };
+  //   await render(hbs`
+  //     <BasicDropdownTrigger tabindex={{false}} @dropdown={{this.dropdown}}>Click me</BasicDropdownTrigger>
+  //   `);
```

Comment:
Could use some feedback on this one - This is the only test I haven't been able to get passing.

When `tabindex={{false}}`, that is presented as `null` once we're inside the modifier (null indicates an attribute is not set). This is also what the modifier uses to determine if it should apply tabindex="0" (and auto-manage it when disabled)

In other words, there's no way for the modifier to know the difference between
`<BasicDropdownTrigger tabindex={{false}} ... />`
and
`<BasicDropdownTrigger ... />`

Any ideas?